app/views/shipit/stacks/settings.html.erb

```diff
@@ -80,3 +90,11 @@
 
   </section>
 </div>
+<script type="text/javascript">
+function fetchCCMenuURL() {
+  $.ajax({url: "<%= ccmenu_url_url(stack_id: @stack.to_param) %>", success: function(result){
+    console.log(result);
+    $("#ccmenu-url").html("<input type='text' disabled value='" + result.ccmenu_url + "'/>");
```

Comment:
> As for the URL, you can store it in the DOM, so that the script doesn't have to craft it.

Not sure I understand here - do you mean generate the url on page load and avoid the ajax entirely? We would probably create client unnecessarily then so I assume you mean something else?